### PR TITLE
Completed HTML forms.

### PR DESCRIPTION
**Pull request by K-Karneboge**
image

I created the form visually, but am not sure how to check the functionality of the form.

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <option value="powerful">Because it's powerful.</option>
+      </select>
+    </div>
+    <br>
+    <div>
+      <label for="favorite">Favorite Language (choose one)</label>
+      <div>
+        <input type="radio" name="favorite" value="HTML">HTML
+        <input type="radio" name="favorite" value="CSS">CSS
+        <input type="radio" name="favorite" value="JavaScript">JavaScript
```
Comment (BrettAlbright) [inline html-forms/index.html]: make sure to include a label for each of these inputs

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looking great Kevin, small update needed!

File: html-forms/index.html (hunk 1 above)
Comment (K-Karneboge) [reply html-forms/index.html]: image
Is this syntax acceptable if the label is short?

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**